src/cmd_index.cpp

```diff
@@ -86,9 +86,33 @@ static void set_field( library & lib, std::string const & name, std::string cons
     lib[ name ].front() = value;
 }
 
+static bool has_meta_file( std::string const & path )
+{
+    return fs_exists( path + "/meta/libraries.json" ) ||
+           fs_exists( path + "/.boost" );
+}
+
+static std::string find_meta_file( std::string const & path )
+{
+    bool meta_libraries = fs_exists(path + "/meta/libraries.json");
+    bool dot_boost = fs_exists(path + "/.boost");
+
+    if( meta_libraries && dot_boost )
```

Comment:
Probably better to just prefer `.boost` if it exists, regardless of whether or not `meta/libraries.json` exists.